### ISSUE TITLE
Issue with setting up the app: Timed out while connecting to butlerd

### ISSUE THREAD

**Issue by JessicaPayne** [itchio/itch.io#1012]
The app was working fine a couple days ago. I was downloading some games, like normal. I opened a game and after a few minutes tried to close it. I had to restart my computer since it wouldn't let me. When the restart finished, I tried to open the app again. Instead of it working like before, it keeps telling me 'There was a problem during setup: Time out while connecting to butlerd'. I've refreshed multiple times, retried the setup, and reinstalled the app twice. Nothing has yet to change.

**Comment by leafo** [comment]
Do you have a virus scanner, firewall, or something similar? If you have any security software please temporarily disable it and try again. Thanks

**Comment by leafo** [comment]
Just checking in again to see if you resolved the issue. 99% of the time, problems like this are because of virus scanning software that is blocking some part of our software because of a false positive.

You can work around this issue by temporary disabling your virus scanner or finding a different virus scanner.

Thanks

**Comment by BellaSatine** [comment]
I'm having the same problem  T^T And I did the same thing to try to fix the issue and anything I tried to do was worthless T^T

**Comment by fasterthanlime** [comment]
> I'm having the same problem T^T And I did the same thing to try to fix the issue and anything I tried to do was worthless T^T

Can you detail the exact steps that happened like @JessicaPayne did?

Can you attach the file at `%APPDATA%\itch\logs\itch.txt` ?

**Comment by BellaSatine** [comment]
There was a problem during setup: Timed out while connecting to butlerd :  Plus it also said that the anti virus may interfere with itch O_O  I tried closing the antivirus software. but to no avail T^T Is it a bug of some sort or something else??? I really enjoy that website and this is all new to me with not being able to be on that site T^T And I'm sorry if that isn't enough information, but I'm not exactly a tech wiz around computers XD I just know the basics O_O

**Comment by fasterthanlime** [comment]
> but I'm not exactly a tech wiz around computers XD

That's okay, that's exactly why we have an app :) Can you tell me which Antivirus you're using?

**Comment by BellaSatine** [comment]
I use Avira antivirus :) I've always been using that antivirus tho O_O Is
it on itch's end or mine???  I've never had this happen with itch  before
XDD

**Comment by fasterthanlime** [comment]
> I use Avira antivirus :) I've always been using that antivirus tho O_O Is it on itch's end or mine??? I've never had this happen with itch before XDD

All Antivirus software is more or less useless and breaks legitimate applications all the time.

So it's on them for making a bad product, but it's not like they're going to go ahead and stop breaking everyone's stuff - it's on us application developers to find and ship workarounds for the outrageous stuff they keep pulling.

TL;DR it's probably their fault, but it's definitely my problem.

I was asking so that I can try to reproduce the issue, but of course from one computer to another, the problems aren't the same, even with the same Antivirus..
